src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-user-adoption-metrics-performance.mdx

```diff
@@ -78,78 +55,20 @@ Each version includes several metrics to analyze performance, user adoption, and
   >
     Key technical performance indicators include stability, network health, other app or server problems, and memory usage:
 
-    * To track stability, the UI shows the **Crash rate** as the percentage of all app sessions that crashed.
-    * To understand whether networking problems are originating from the app, network, or server, the UI shows the percentage of server responses that returned an HTTP error code (**HTTP errors**) and the percentage of network calls that failed to receive a response (**Network failure**).
-    * To identify other problems from the app or server, the UI shows the average number of network requests made per app session (**Requests per session**) and the average server **Response time** as seen from the app’s perspective.
-    * To compare improvements or problems with **Memory** usage, the first four versions include metrics indicating better (green) or worse (red) performance than the previous version's.
+    * **Crash rates:** the percentage of all app sessions that crashed
+    * **HTTP errors:**  HTTP errors are HTTP requests that have a status code greater than 400
```

Comment:
I'm not sure what "greater than 400" means here.

Reply:
It's for HTTP errors: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes-- I'm not sure how to clarify this any differently?

Reply:
I think it's the "greater than 400" that's throwing me. Maybe it's better to say something like "status codes between 400 and 499"? or "status codes from 401-499"

Reply:
Done!